### PR TITLE
Schedule Activity Event

### PR DESCRIPTION
**Pull request by mbouadeus**
Completed schedule activity event servlet.
https://screenshot.googleplex.com/25VqGpER00p.png

Note: We found out that to solve the routing problem, you also have to do mvn clean install on the project after the web.xml file has been added.

### CODE REVIEW

**Review by bhargaviks — CHANGES_REQUESTED** [review]
Don't we have to gitignore webapp folder since each time we commit we will be getting the build files, which is unnecessary? Thoughts?

Also, I don't visually see a new line at EOF in some files, but there is no alert ('-' inside a red circle) while reviewing. Why is that happening?

**Comment by mbouadeus** [comment]
I didn't add the webapp folder in .gitignore because it has the WEB-INF folder inside. That folder is crucial for the project and it has to reside in webapp. Instead I'm going to see if I can ignore everything in the webapp folder except the WEB-INF folder.

Regarding new lines at the end of files, github will not show you the new lines from my understanding. That's why it puts the red symbol. It's to indicate that the new line isn't there.

**Comment by bhargaviks** [comment]
Oh really. During the first few weeks of the internship, when I did have a new line it would show. Sounds weird. Alright, as long as it's there, it is cool. :) I will approve your pull request now, and assume that you will address the gitignore stuff in the next pull request?

**Review by bhargaviks — APPROVED** [review]

**Comment by mbouadeus** [comment]
Just removed the all the stuff in webapp except WEB-INF and removed the build files.

File: src/main/java/com/google/sps/data/CalendarUtility.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+public final class CalendarUtility {
+  /**
+   * Global instance of the {@link DataStoreFactory}. The best practice is to make it a single
+   * globally shared instance across your application.
+  */
+  private static final AppEngineDataStoreFactory DATA_STORE_FACTORY =
+    AppEngineDataStoreFactory.getDefaultInstance();
+  
+  /** Global instance of the HTTP transport. */
+  static final HttpTransport HTTP_TRANSPORT = new UrlFetchTransport();
```
Comment (pengxianzhe) [inline src/main/java/com/google/sps/data/CalendarUtility.java:37]: Is there a reason for HTTP_TRANSPORT and JSON_FACTORY to be not private?

**Review by isaacwdavis — APPROVED** [review]

**Review by pengxianzhe — APPROVED** [review]

File: src/main/java/com/google/sps/data/CalendarUtility.java (hunk 1 above)
Comment (mbouadeus) [reply src/main/java/com/google/sps/data/CalendarUtility.java:37]: You are right. I will add it in the next pull request.

**Pull request merged**